docs/workloads/azure/data/getting_started/dev_quickstart_data_azure.md

```diff
@@ -55,7 +55,7 @@ To run the E2E tests, you need to set up the following environment variables:
 - `AZURE_STORAGE_ACCOUNT_NAME`
```

Comment:
Env variables should be updated:
AZURE_SUBSCRIPTION_ID
AZURE_RESOURCE_GROUP_NAME
AZURE_DATA_FACTORY_NAME
AZURE_REGION_NAME
AZURE_STORAGE_ACCOUNT_NAME

Reply:
Resolve in latest push.